packages/teams-js/src/public/pages.ts

```diff
@@ -232,7 +233,7 @@ export namespace pages {
    */
   export function registerFullScreenHandler(handler: fullScreenChangeFunctionType): void {
     registerHandlerHelper('fullScreenChange', handler, [], () => {
-      if (handler && !isSupported()) {
+      if (!isNullOrUndefined(handler) && !isSupported()) {
```

Comment:
>!isNullOrUndefined(handler) [](http://example.com/codeflow?start=10&length=27)

Using an explicit `undefined` and `null` check rather than the implicit `if (handler)` check makes it clear to the compiler of the intent (without that check, the compiler would question whether the intent was calling the `handler()` function)